### ISSUE TITLE
Error with any peacock command attempt when ssh user doesnt have write permissions 

### ISSUE THREAD

**Issue by userthirtytwo** [johnpapa/vscode-peacock#475]
First off, peacock is awesome and I love it!  thanks for your hard work!

**Is your feature request related to a problem? Please describe.**
![image](https://user-images.githubusercontent.com/77980817/138454797-f3bc573d-c5bd-4b04-a396-ca0f416e48b3.png)
Wouldn't call it a problem really, but I just noticed that since peacock seems to write to the current directory in the vsc window, any denial of write access causes peacock to error out.  (at least I think this is what is happening- also I should mention I'm mostly just using vscode for ssh).
This doesnt happen when ssh'ing into a session with a root user.  I just happened to notice this when I spun up some ubuntu vms for testing and peacock wont work because I didn't select the home directory of the user as a base directory.

**Describe the solution you'd like**
In short, I just want the pretty colors so it's easy to distinguish hosts.  It would be ok if the colors arent saved and auto-applied next time I login...

**Describe alternatives you've considered**
Using a vscode ssh client without pretty colors (pretty grim at this point)

Thanks again!

**Comment by johnpapa** [comment]
correct, it write to the file locally.

closing as this a fundamental way the tool works given the API for vs code.
